packages/patternfly-4/react-core/src/components/Breadcrumb/__snapshots__/Breadcrumb.test.tsx.snap

```diff
@@ -13,38 +12,28 @@ exports[`Breadcrumb component should render breadcrumb with aria-label 1`] = `
 
 exports[`Breadcrumb component should render breadcrumb with children 1`] = `
 <nav
-  aria-label="Breadcrumb"
   className="pf-c-breadcrumb"
 >
   <ol
     className="pf-c-breadcrumb__list"
   >
-    <BreadcrumbItem
-      className=""
-      component="a"
-      isActive={false}
-      target={null}
+    <Component
```

Comment:
I don't understand why this is rendering as `<Component>` now?

Reply:
It's rendering `component: Component = 'a' as any` in the shallow render now since it doesn't use defaultProps.

Reply:
If we do mount does it render 'a'?  We need to look into this further.  Rendering as `Component` here devalues the usefulness of the snapshot test in my opinion.

Reply:
I opened an issue to track this

Reply:
Awesome, thanks! I dug into this issue a little bit, and it seemed more complicated than I originally thought.